openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -812,43 +812,46 @@ def _assign_aromaticity_and_stereo_from_3d(self, offmol):
         return offmol_w_stereo_and_aro
 
     def enumerate_protomers(
-        self, molecule: "Molecule", max_states: int = 10
+        self,
+        molecule: "Molecule",
+        max_states: int = 0,
     ) -> list["Molecule"]:
         """
-        Enumerate the formal charges of a molecule to generate different protomoers.
+        Enumerate the formal charges of a molecule to generate different protomers.
 
         Parameters
         ----------
-        molecule: openff.toolkit.topology.Molecule
+        molecule
             The molecule whose state we should enumerate
 
-        max_states: int optional, default=10,
-            The maximum number of protomer states to be returned.
+        max_states
+            The maximum number of protomer states to be returned. If 0, the default,
+            attempt to return all protomers. If set to a non-zero number, the input molecule
+            is not guaranteed to be included in the returned list.
```

Comment:
Even if set to 0, the input isn't guaranteed to be returned (Ex. `NH2-`)
```suggestion
        max_states
            The maximum number of protomer states to be returned. If 0, the default,
            attempt to return all protomers. 
```